Source/Krypton Components/Krypton.Ribbon/Palette/PaletteRedirectRibbonAeroOverride.cs

```diff
@@ -50,7 +50,7 @@ public override Color GetContentShortTextColor1(PaletteContentStyle style, Palet
             {
                 // If the ribbon is showing in office 2010 style and using glass
                 if (_ribbon.CaptionArea.DrawCaptionOnComposition
-                    && _ribbon.RibbonShape is PaletteRibbonShape.Office2010 or PaletteRibbonShape.Office2013 or PaletteRibbonShape.Microsoft365)
+                    && _ribbon.RibbonShape is PaletteRibbonShape.Office2010 or PaletteRibbonShape.VisualStudio2010 or PaletteRibbonShape.VisualStudio2010 or PaletteRibbonShape.Office2013 or PaletteRibbonShape.Microsoft365 or PaletteRibbonShape.VisualStudio)
```

Comment:
Condition is used repeatedly, so probably best as a function

Reply:
Now is its own function